### DISCUSSION TITLE
Tojan in tabby_x86_64-windows-msvc-cuda117.zip.

### DISCUSSION THREAD

**Discussion by danyboh** [TabbyML/tabby#3632] [Q&A] [answered]
Just downloaded latest release, and windows defender blocked it saying it has a trojan. Is it expected?

**Comment by icycodes** [comment, marked as answer]
No, it is not expected. I just tested downloading `tabby_x86_64-windows-msvc-cuda117.zip` from the v0.22.0 release. The SHA256 checksum is `434AD39623039ECFFF3C9BCFBD62F925B638E015D420CEB8E8BE0EBE24B92FCE`. It can be downloaded and unzipped normally without being blocked by Windows Defender.

You might see a warning about running an unrecognized app or accessing the network when running `tabby.exe`, but it should not be flagged as a trojan.
